src/components/ContentSidebar/DetailsSidebar.js

```diff
@@ -70,6 +74,8 @@ const DetailsSidebar = ({
                         owner={getProp(file, 'owned_by.name')}
                         size={getFileSize(file.size, intl.locale)}
                         uploader={getProp(file, 'created_by.name')}
+                        onDescriptionChange={onDescriptionChange}
```

Comment:
Do the can_rename check here

Reply:
why exactly can't it be inline? `false && function` would return false, and `true && function` would return true. Is it because we would want to pass in either a function or null into ItemProperties?

Reply:
onDescriptionChange type is function, and you are sending it a boolean